### DISCUSSION TITLE
modello telecomando rolling code per contatto pulito

### DISCUSSION THREAD

**Discussion by andreablu** [indomus/forum#2283] [GENERALE - Discussioni generiche]
Ciao a tutti,

mi potreste fornire dei modelli di telecomandi universali facilmente integrabili con shelly? nel senso per prendere il contatto del pulsante.

al momento ho un NICE INIT 2 e vorrei clonarlo con un modello universale e poi da li seguire la guida per domotizzarlo fisicamente.

Grazie in anticipo

Andrea

**Comment by KC-inDomus** [comment]
non ho capito la domanda
